### DISCUSSION TITLE
Private domain alias not forwarding email

### DISCUSSION THREAD

**Discussion by pangag** [simple-login/app#666] [Q&A]
I just had my domain setup and followed the instructions ang they're all green (all verified). But I am not able to receive email. I am able to send email via SimpleLogin dashboard.

**Comment by nguyenkims** [comment]
Can you paste Postfix log (usually in `/var/log/mail.log`) and `sl-email` container log here as they can contain the reason why?

**Comment by trikotret** [comment]
make sure you checked catch all after adding domain.
